app/services/midata_service.rb

```diff
@@ -32,11 +37,21 @@ def fetch_camp_unit_data_hierarchy(root_url, root: true)
 
     return root_data if children_ids.nil? && !root
 
+    [root ? nil : root_data, children_data(children_urls(children_ids, root_data))].flatten.compact
```

Comment:
@lukaseisenring i guess that was the reason that the root camps always got imported.